deploy/acm-policies/50-GENERATED-hs-mgmt-route-monitor-operator.Policy.yaml

```diff
@@ -32,6 +32,7 @@ spec:
                         metadata:
                             name: api
                         spec:
+                            domainRef: hcp
```

Comment:
This doesn't feel like it should be part of this PR.

Reply:
This came out of the make run and I was not sure what to do with it.
I am happy to remove it though.

Reply:
+1 for removal, thanks. I'm not sure why it's being added when generated locally, but as it's a change not related to the intent of the PR I would be more comfortable with it not going out.

Reply:
Thank you. That sounds like the best plan. I will revert that.